openreview/conference/templates/areachairWebfield.js

```diff
@@ -600,6 +612,18 @@ var buildTableRow = function(note, reviewerIds, completedReviews, metaReview) {
   return [cellCheck, cell0, cell1, cell2, cell3, cell4, cell5];
 };
 
+var findNextAnonGroupNumber = function(paperNumber){
```

Comment:
This function is seems overly complex... basically you just want to find the next available reviewer number from the reviewers map right when there might be gaps in the keys? So instead of searching the whole list each time why don't you just check if `i === paperReviewerNums[i]` and if not `return i`? If you get through the whole list `return i + 1`

Reply:
Agreed. The function has been refactored as per suggestion.